openwisp_monitoring/monitoring/templates/monitoring/paritals/chart.html

```diff
@@ -4,13 +4,14 @@
     <span id="monitoring-timeseries-api-url" data-value="{{ api_url }}"></span>
     <span id="monitoring-timeseries-original-key" data-value="{{ original.key }}"></span>
     <span id="monitoring-timeseries-default-time" data-value="{{ default_time }}"></span>
+    <script type="text/javascript" src="https://cdn.jsdelivr.net/momentjs/latest/moment.min.js"></script>
+    <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/daterangepicker/daterangepicker.min.js"></script>
+    <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/daterangepicker/daterangepicker.css" />
```

Comment:
This is not the correct way to load scripts on the dashboard page because the scripts will get loaded before jQuery is initialised. It appears to me that openwisp-utils does not allow defining additional scripts for the dashboard at the moment. 

For now, you can download these files and serve them from Django itself. `openwisp_monitoring/device/static/monitoring/` would be the directory to store this files. Then you will need to add them in the following code so it will be loaded in the dashboard page. 


https://github.com/openwisp/openwisp-monitoring/blob/76942a31e0d655a246eb846c94434fc53e14795c/openwisp_monitoring/device/apps.py#L266-L281